source/index.txt

```diff
@@ -14,9 +14,10 @@ MongoDB Shell
 
 .. include:: /includes/admonitions/fact-mdb-shell-beta.rst
 
-The |mdb-shell|, ``mongosh``, is a fully functional JavaScript
-environment for interacting with MongoDB deployments. You can use the
-|mdb-shell| to test queries and operations directly with your database.
+The |mdb-shell|, ``mongosh``, is a fully functional JavaScript and 
+Node.js v14.x |repl| environment for interacting with MongoDB 
+deployments. You can use the|mdb-shell| to test queries and operations 
```

Comment:
```suggestion
Node.js 14.x |repl| environment for interacting with MongoDB 
deployments. You can use the |mdb-shell| to test queries and operations 
```

The second bit is just a typo, I think? It doesn’t look quite right on the generated page to me. The first bit is just Node.js’s own stylistic preference for how Node.js versions should be spelled out in prose, feel free to ignore it, especially if there are rules for the mongodb documentation that contradict this. :slightly_smiling_face: